urdf/urdfdom_compatibility.h.in

```diff
@@ -69,16 +69,25 @@ URDF_TYPEDEF_CLASS_POINTER(Sphere);
 URDF_TYPEDEF_CLASS_POINTER(Visual);
 
 URDF_TYPEDEF_CLASS_POINTER(ModelInterface);
+
+typedef boost::shared_ptr<void> VoidSharedPtr;
```

Comment:
..we cloud export a define in the `urdf/urdfdom_compatibility.h.in`, would that be ok?